### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by jesusmoraleda**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [X ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [X ] Your additions are ordered alphabetically
- [X ] Your submission has a useful description
- [X ] The description does not end with punctuation
- [X ] Each table column should be padded with one space on either side
- [X ] You have searched the repository for any relevant issues or pull requests
- [ X] Any category you are creating has the minimum requirement of 3 items
- [X ] All changes have been [squashed][squash-link] into a single commit

Hola!
Somos un grupo de estudiantes de la Universidad Complutense de Madrid (UCM) y estamos
haciendo una campaña de concienciación sobre el uso de licencias.

Razones por las que usar una licencia:

- Es FÁCIL de usar: escoge la licencia, copia el texto genérico y listo! Ya puedes usarla en tus proyectos. En el siguiente [link](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository) puedes encontrar más información:
- LIMITAS qué usos pueden tener las personas para usar tu contenido, de esta manera evitas problemas de copias y plagios 
- Compartir es poder! Las licencias libres te dan credito en todo momento permitiendo que otros usen tu trabajo bajo TUS condiciones.
En el siguiente [video](https://www.youtube.com/watch?v=OnmWFxlG2GA) se explican los tipos de licencias para que puedas escoger una que se adapte a tus necesidades

Hemos elegido la licencia MIT para este pull request. En este [link](https://choosealicense.com/licenses/mit/) puedes encontrar mas informacion sobre esta licencia

Gracias por tu atención! Esperamos que te haya servido de ayuda

------------------------------------------------------------------------------------------------------------------
Hello!
We are a group of students of Universidad Complutense de Madrid (UCM) and we are working on an awareness campaign about types and uses of licences.

There are many reasons to use licences in your code:

- It's EASY to use: choose the licence, copy the generic text and ready! Now you can use it in your projects You can found more information in the following [link](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/licensing-a-repository ) 

- You LIMIT the uses of your code, in this way you will avoid possible legal problems about copies and plagiarism
- Share is power! The freeware licences give to you credit all the time. Allow third people use it with YOUR conditions

The following [youtube link](https://www.youtube.com/watch?v=OnmWFxlG2GA) show the existing types of licences to choose one which adapt better to your needs

We have chosen MIT license for this pull request. In this [link](https://choosealicense.com/licenses/mit/), you can get more information about this license

Thank you a lot for your attention! We hope you the contents has been helpful

### CODE REVIEW

**Comment by marekdano** [comment]
I'm happy to have the MIT licence for this project. What do you think @yannbertrand, @matheusfelipeog?

**Comment by matheusfelipeog** [comment]
> I'm happy to have the MIT licence for this project. What do you think @yannbertrand, @matheusfelipeog?

This is perfect for me

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019  public-apis
```
Comment (PawelBorkar) [inline LICENSE:3]: Please bump the year to 2021.

**Review by PawelBorkar — COMMENTED** [review]
@marekdano I'm fine with this too.

**Comment by yannbertrand** [comment]
Agreed

**Comment by marekdano** [comment]
@jesusmoraleda can you please update the year to 2021 as @PawelBorkar suggested, please

**Comment by PawelBorkar** [comment]
As discussed and with the approval of @marekdano and @matheusfelipeog I have added the MIT License to the project since the author of this PR hasn't been active from couple of months or more. 

[License](https://github.com/public-apis/public-apis/commit/fb5d02be18d464553b57ebd68ca9259635b9a7e3)

**Pull request closed without merging**